src/device.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+use super::*;
+use std::sync::atomic::AtomicUsize;
+use std::sync::Arc;
+use vm_memory::GuestMemory;
+use vmm_sys_util::EventFd;
+
+/// Trait for virtio devices to be driven by a virtio transport.
+///
+/// The lifecycle of a virtio device is to be moved to a virtio transport, which will then query the
```

Comment:
Please keep the comments under 100 chars/line.

Reply:
Fixed.